meetings/2023-04-19.md

```diff
@@ -0,0 +1,131 @@
+* EP: Question to PA: Looking at the deliverables section in the [charter](http://htmlpreview.github.io/?https://github.com/solid/solid-wg-charter/blob/12a9e08ff5876e24c0f1b7c0f180d3e1bc5b6efa/charter/index.html) how specific should we be?
+* EP: Currently the deliverables section matches the organization of the specifications in the CG. Isn't this over-specific for the charter?
+* PA: I think it is good to point to existing work that is in scope. Which is the goal of these pointers in the charter. There is a strong dependency between the specification and these references. I don't have a strong opinion on whether this should be in the scope or deliverables section.
+* SC: Just a reminder that they were introduced based on prior observations on other charters / formal objections (e.g. [Devices and Sensors WG](https://www.w3.org/groups/wg/das), Web Application WG). See past discussion: https://github.com/solid/specification/blob/main/meetings/2023-04-12.md#clarify-normative-tentative-new-deliverables. The general idea is that if it is not sufficiently incubated, it should be listed as an in-progress or tentative specification that the WG could consider as a deliverable.
+* EP: We could have a list of more adopted drafts, e.g., WAC, ACP, etc. We cannot expect the final protocol to have the functionality, if it is not in the scope of the WG.
+* SC: Suggestion that PA review the PR and validates whether this is along the lines of the W3C guidelines and is sufficient to wrap up the deliverable.
+* PA: Yes, I will review this PR and try to get feedback from other W3C people about how to address this. I like the idea to list them as other "adopted" drafts, however this might indicate a stronger commitment. 
+* EP: Since you mentioned N3, personally I think the protocol would be in a good state even if we don't depend on this as we have other approaches for this. We should also clarify dependency on WebID, as this is a relatively old draft and perhaps consider adopting it in the WG as a normative dependency.
+* SC: As these six specifications (Solid-OIDC, WAC, ACP, Notifications, WebID, N3) are of a similar status, we should leave the flexibility to adopt them if the WG deems these relevant but we haven't made a commitment. All we have committed to is that they are a tentative deliverable and we may adopt them.
+* TT: I have found a [relevant thread from a TAG discussion (2012)](https://lists.w3.org/Archives/Public/www-tag/2012Apr/0190.html), which mentions there is no strict rule but there is guidance in transition process which effectively requires that the reference should have a status that is no further than one step behind the specification. Otherwise the reference should be informative instead of normative.
+* EP: If something is a informative reference is omitted by an implementation, then an implementation ignoring it would still be conformant?
```

Comment:
```suggestion
* EP: If something in an informative reference is omitted by an implementation, then an implementation ignoring it would still be conformant?
```